File: snippets/proxy/create-participants/create-participants.py
# Download the helper library from https://www.twilio.com/docs/python/install
from twilio.rest import Client

# Your Account SID and Auth Token from twilio.com/console
account_sid = 'ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
auth_token = 'xxxxxxxxxxxxxxxxxxxxxxxxxxx'
client = Client(account_sid, auth_token)

proxy_sid = 'KSxxxxxxxxxxxxxxxxxxxxxxxxxx'
session_sid = 'KCxxxxxxxxxxxxxxxxxxxxxxxxxx'

participant = client.proxy \
                    .services(proxy_sid) \
                    .sessions(session_sid) \
                    .participants \
                    .create(friendly_name='Alice', identifier='+15554441212')

print(participant.sid)

File: snippets/proxy/create-participants/create-participants.java
import com.twilio.Twilio;
import com.twilio.rest.proxy.v1.service.session.Participant;

public class create_participants {

    private static final String ACCOUNT_SID = System.getenv("ACCOUNT_SID");
    private static final String AUTH_TOKEN = System.getenv("AUTH_TOKEN");

    public static void main(String[] args) {
        Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

        // https://www.twilio.com/docs/proxy/api/participant
        Participant participant = Participant.creator(
                System.getenv("PROXY_SERVICE_SID"),
                "KCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
                "+15558675310"
        )
                .setFriendlyName("Alice").create();

        System.out.println("+ Participant added to Proxy service session, ProxyIdentifier: " + participant.getSid());
    }
}
